Foundation/NSJSONSerialization.swift

```diff
@@ -281,6 +291,21 @@ internal extension JSONSerialization {
     }
 }
 
+private protocol AnyOptional {
+    var value: Any? { get }
+}
+
+extension Optional : AnyOptional {
+    public var value: Any? {
+        switch self {
+        case .none:
+            return nil
+        case .some(let x):
+            return x
+        }
+    }
+}
+
```

Comment:
Ok, to match our naming conventions I think this protocol should at the very least start with an `_`. The way it is named now it really looks like a standard library API and I think it's a bit confusing.